conda_content_trust/root_signing.py

```diff
@@ -173,7 +173,7 @@ def sign_via_gpg(data_to_sign, gpg_key_fingerprint, include_fingerprint=False):
     # care about that as much -- we want to use the raw ed25519 public key
     # value to refer to the key in a manner consistent with the way we refer to
     # non-GPG (non-OpenPGP) keys.
-    keyval = fetch_keyval_from_gpg(gpg_key_fingerprint)
+    _ = fetch_keyval_from_gpg(gpg_key_fingerprint)
```

Comment:
Is this here because it was included in the return value in the past? Is it trying to throw an exception if the key does not exist? Or can it be removed? See L192

Reply:
keyval is an unused variable ... this can either 1. be removed due to not required or 2. it relies on a side effect = exception raised (there should be then a test),  ... requires a deeper look... intend of the PR was remove the lint ignores ... will see what can be done here.